modules/odc-creating-apiserversource.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * applications/application_life_cycle_management/odc-knative-event-sources.adoc
+
+[id="odc-creating-apiserversource_{context}"]
+= Creating an ApiServerSource event source using the Developer perspective
+
+.Procedure
+
+. Navigate to the **Add** page.
++
+image::event-sources-odc.png[Add an Event Source in the Developer perspective]
+. Select **Event Sources** -> **ApiServerSource**.
++
+image::apiserversource-odc.png[Create an ApiServerSource]
+. Configure the **ApiServerSource** settings:
++
+image::apiserver-odc-config-1.png[Configuring the ApiServerSource]
++
+.. Enter `sources.knative.dev/v1alpha1` as the **APIVERSION**, and `ApiServerSource` as the **KIND**.
+// TODO: make sure version is right across all serverless docs, if this needs to be updated, update in `kn` docs too.
+// .. What does 'mode' do?
+.. Select the **Service Account Name** for the service account that you created for event sources.
+.. Select the targeted Knative service from the dropdown menu in **Sink** -> **Knative Service**.
+// .. Add details about the image:
+// General -> Application Name, Name
+// image::apiserver-odc-config-2.png[Configuring the ApiServerSource]
+. Click **Create**.
+
+.Verification steps
+
+. After you have created the ApiServerSource, you will see it connected to the service it is sinked to in the **Topology** view.
```

Comment:
Can even show sidebar on click to show associated deployments and pods for `ApiServerSource` are up and running.

<img width="1499" alt="Screenshot 2020-06-16 at 12 39 30 PM" src="https://user-images.githubusercontent.com/5129024/84742676-763c2b80-afce-11ea-8bbf-67030809b6b1.png">